docs/operate/relayer/run-relayer.mdx

```diff
@@ -50,8 +18,6 @@ Running a Relayer requires the following:
 
 ## Guide
 
-We strongly encourage you to follow the [local agents guide](../../guides/deploy-hyperlane-local-agents.mdx) to understand how to configure and run the Relayer locally.
```

Comment:
I removed it because it was mentioned on the top as well, so it seemed like a repeat too soon. (its mentioned just under 'Run a Relayer' header)

But now that you bought it to my attention I think i should have left it, so I re-added (inside the info section). 
![Screenshot 2024-10-17 at 09 03 26](https://github.com/user-attachments/assets/7b3bf612-7011-44e5-8521-b86b5e652923)

More info + tips are better. Hope this works.